docs/content.zh/docs/connectors/datastream/firehose.md

```diff
@@ -0,0 +1,171 @@
+sinkProperties.put(AWSConfigConstants.AWS_SECRET_ACCESS_KEY, "aws_secret_access_key")
+
+val kdfSink =
+    KinesisFirehoseSink.<String>builder()
+        .setKinesisClientProperties(sinkProperties)    // Required
+        .setElementConverter(elementConverter)         // Required
+        .setDeliveryStreamName("your-stream-name")     // Required
+        .setFailOnError(false)                         // Optional
+        .setMaxBatchSize(500)                          // Optional
+        .setMaxInFlightRequests(50)                    // Optional
+        .setMaxBufferedRequests(10_000)                // Optional
+        .setMaxBatchSizeInBytes(4 * 1024 * 1024)       // Optional
+        .setMaxTimeInBufferMS(5000)                    // Optional
+        .setMaxRecordSizeInBytes(1000 * 1024)      // Optional
+        .build()
+
+flinkStream.sinkTo(kdfSink)
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Configurations
+
+Flink's Cassandra sink is created by using the static builder `KinesisFirehoseSink.<String>builder()`.
+
+1. __setKinesisClientProperties(Properties sinkProperties)__
+    * Required.
+    * Supplies credentials, region and other parameter to the firehose client.
+2. __setElementConverter(KinesisFirehoseSinkElementConverter elementConverter)__
+    * Required.
+    * Supplies a serialization schema to the output. May be built using the following builder `KinesisFirehoseSinkElementConverter.<String>builder()` as per the example.
+3. __setDeliveryStreamName(String deliveryStreamName)__
+    * Required.
+    * Name of the delivery stream to sink to.
+4. _setFailOnError(boolean failOnError)_
+    * Optional. Default: `false`.
+    * Whether failed requests to write records to Firehose are treated as fatal exceptions in the sink.
+5. _setMaxBatchSize(int maxBatchSize)_
+    * Optional. Default: `500`.
+    * Maximum size of a batch to write to Firehose.
+6. _setMaxInFlightRequests(int maxInFlightRequests)_
+    * Optional. Default: `50`.
+    * The maximum number of in flight requests allowed before the sink applies backpressure.
+7. _setMaxBufferedRequests(int maxBufferedRequests)_
+    * Optional. Default: `10_000`.
+    * The maximum number of records that may be buffered in the sink before backpressure is applied. 
+8. _setMaxBatchSizeInBytes(int maxBatchSizeInBytes)_
+    * Optional. Default: `4 * 1024 * 1024`.
+    * The maximum size (in bytes) a batch may become. All batches sent will be smaller than or equal to this size.
+9. _setMaxTimeInBufferMS(int maxTimeInBufferMS)_
+    * Optional. Default: `5000`.
+    * The maximum time a record may stay in the sink before being flushed.
+10. _setMaxRecordSizeInBytes(int maxRecordSizeInBytes)_
+    * Optional. Default: `1000 * 1024`.
+    * The maximum record size that the sink will accept, records larger than this will be automatically rejected.
+11. _build()_
+    * Constructs and returns the Firehose sink.
+
+
+## Using Custom Kinesis Endpoints
```

Comment:
Any reference to "Kinesis" here feels out of place. Typically when people say "Kinesis" they mean KDS